src/components/expeditions/ExpeditionsModal.tsx

```diff
@@ -45,7 +45,7 @@ export default function ExpeditionsModal({ onDismiss }: { onDismiss: () => void
           </Row>
           <Row>
             <Text>
-              Follow Swapr <ExternalLink href="https://twitter.com/swapreth">Twitter</ExternalLink> or{' '}
+              Follow Swapr <ExternalLink href="https://twitter.com/swapr_dapp">Twitter</ExternalLink> or{' '}
```

Comment:
Since we are changing the link maybe we can update it to X instead of twitter.

Reply:
Twitter and X work currently so I'm ok with both